08.Packages.md

```diff
@@ -0,0 +1,12 @@
+# Packages
```

Comment:
Update to whatever we decide to go with on the Welcome page title (also update file name)